src/resource/tiles/schemas/tile.schema.ts

```diff
@@ -0,0 +1,32 @@
+import { Schema, Prop, SchemaFactory } from "@nestjs/mongoose";
+import { Document } from "mongoose";
+
+/* maps our cat class to our mongoDB collection of the same name */
+@Schema()
+export class Tile extends Document {
```

Comment:
which column is primary key?

Reply:
no, naming cannot be primary key

Reply:
_id should be a primary key, Mongo DB will generate _id by default.